lib/tasks/factory_girl.rake

```diff
@@ -1,9 +1,11 @@
+require_relative '../../spec/support/external_request'
 namespace :factory_girl do
   desc "Verify that all FactoryGirl factories are valid"
   task lint: :environment do
     if Rails.env.test?
       begin
         DatabaseCleaner.start
+        mock_commercial_request
```

Comment:
:+1: Although I would put this in a separate commit as it is currently wrong and it is unrelated to the Travis error